src/Tasks/Microsoft.Common.CrossTargeting.targets

```diff
@@ -248,4 +248,9 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
   <Import Project="$(DirectoryBuildTargetsPath)" Condition="'$(ImportDirectoryBuildTargets)' == 'true' and exists('$(DirectoryBuildTargetsPath)')"/>
 
+  <PropertyGroup>
```

Comment:
Why do you need this PropertyGroup? Is the property used elsewhere? ('' != 'true', so this shouldn't affect the Import on the next line.)

Reply:
It's the same pattern used here https://github.com/dotnet/msbuild/pull/9193/files#diff-5407d46dd30ce4031e530c35cc2e0a62a6c96e54cb1def14fb316f351ef92de9 the design expect this usage atm
```bash
msbuild /t:Test /p:UseMSBuildTestInfrastructure=true
```

Reply:
It doesn't look necessary there either, but it isn't really a big deal.